DOCS/man/options.rst

```diff
@@ -4728,6 +4728,21 @@ The following video options are currently all specific to ``--vo=gpu`` and
 
     Currently only relevant for ``--gpu-api=d3d11``.
 
+``--wayland-frame-wait-offset=<-100..3000>``
```

Comment:
inconsistent with the actual option name in the code (did you forget to rename it there?)